scripts/new_version

```diff
@@ -4,13 +4,23 @@
 Make a new release version of BiblioPixel by editing CHANGELIST.md
 
 """
+# TODO: Migrate to https://github.com/olivierlacan/keep-a-changelog
 
 import datetime, os, subprocess, sys
+import re
 
 DRY_RUN = False
 ROOT = os.path.dirname(os.path.dirname(__file__))
 CHANGELIST_FILE = os.path.join(ROOT, 'CHANGELIST.md')
-VERSION_FILE = os.path.join(ROOT, 'bibliopixel', 'VERSION')
+INIT_FILE = os.path.join(ROOT, 'bibliopixel', '__init__.py')
+
+
+with open(INIT_FILE) as fp:
+    try:
+        init = fp.read()
+        __version__ = re.search(r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]', init).group(1)
```

Comment:
This seems fragile.  Certainly, I can't actually evaluate this regular expression for correctness.